R/data_kara.R

```diff
@@ -0,0 +1,49 @@
+#' NBA Elo Ratings Through History
+#' 
+#' The data behind the Elo and RAPTOR ratings for NBA teams used in various stories like this one: 
+#' \url{https://projects.fivethirtyeight.com/complete-history-of-the-nba/#raptors}
+#' 
+#'   @format A data frame with 69636 rows representing every game in the NBA, and 24 variables:
```

Comment:
The extra spacing in front of @format is causing the help file to come out wonky. Could you fix and make sure to read the resulting help file?